compoments/report-repair/summary.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import SummaryList from '../summaryList';
+import Button from '../button';
+const Summary = ({values, changeLinkUrlValues}) => {
+  let [whatIsRepairProblemLink, setWhatIsRepairProblemLink] = useState(null)
+  let [describeRepairProblemLink, setDescribeRepairProblemLink] = useState(null)
+  useEffect(()=>{
+    setWhatIsRepairProblemLink(changeLinkUrlValues[values.repairLocation])
+    setDescribeRepairProblemLink(changeLinkUrlValues[values.repairProblem])
+    console.log('ADDRESS',values.address)
+  })
+  const personalDetailsSummary = [
+    {pageName:'Repair address', value: values.address+ ',' + values.postcode, link: 'postcode'},
```

Comment:
```suggestion
    {pageName:'Repair address', value: values.address.display},
```
displaying the address like this should fix this
<img width="543" alt="Screenshot 2021-12-13 at 09 59 33" src="https://user-images.githubusercontent.com/8051117/145791557-6fa8683b-0825-4969-8a96-47267826345d.png">